### PR TITLE
Initial html

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description

In this pull request I added all the initial HTML elements to the page. Notably, links bootstrap and Material Icons are both included, so the page structure utilizes both. This update also adds a pull-request-template. No styling has been added yet

## Affected areas of application

- index.html
- /docs

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- VSCode LiveServer
- npm

## Relevant Tickets

- Progress towards #/2

## Screenshots

Screen Shot 2020-09-22 at 12 52 37 PM

### CODE REVIEW

**Comment by bretm9** [comment]
I appreciate the screenshot! There are a good amount of commits as well.

File: src/index.html [hunk 1]
```diff
@@ -1,19 +1,133 @@
...
-    <div class="container">
-    <div class="row justify-content-center">
-      <div class="col-3 align-items-center">
-        <img src="./images/turing-logo.png" class="" alt="turing logo" height="100px" width="100px">
-      </div>
-    </div>
-  </div>
-
+    <section class="page-title-section">
+      <div class="container">
```
Comment (bretm9) [inline src/index.html:10]: You may want to refactor this divs into something that utilizes semantic HTML

File: src/index.html [hunk 2]
```diff
@@ -1,19 +1,133 @@
...
-    <h1>what the flippin heck dude</h1>
-    <div class="container">
-    <div class="row justify-content-center">
-      <div class="col-3 align-items-center">
-        <img src="./images/turing-logo.png" class="" alt="turing logo" height="100px" width="100px">
-      </div>
-    </div>
-  </div>
-
+    <section class="page-title-section">
```
Comment (bretm9) [inline src/index.html:9]: Depending on the structuring of the page in future iterations, you may want all your sections contained in a `main` element.

File: docs/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## Description
```
Comment (bretm9) [inline docs/pull_request_template.md:1]: This is a good PR template!

**Review by bretm9 — APPROVED** [review]
All in all, this is a good initial foundation. I'm excited to see how you style the page and organize the layout.

File: src/index.html (hunk 1 above)
Comment (CalebCyphers) [reply src/index.html:10]: Ooh, you're right. I think bootstrap might need the divs to function but I'll look it up. At the very least I'll add a proper tabIndex

File: src/index.html (hunk 2 above)
Comment (CalebCyphers) [reply src/index.html:9]: This is a great point, I totally missed adding the `main` element, thanks!

**Pull request merged**